sdk/communication/azure-communication-sms/tests/test_sms_client_e2e.py

```diff
@@ -9,12 +9,20 @@
 from azure.communication.sms import (
     PhoneNumberIdentifier, SendSmsOptions, SmsClient
 )
+from azure.communication.sms._shared.utils import parse_connection_str
 from _shared.testcase import (
     CommunicationTestCase,
     BodyReplacerProcessor,
     ResponseReplacerProcessor
 )
+from azure.identity import DefaultAzureCredential
 
+class FakeTokenCredential(object):
```

Comment:
i think there is a common class 'FakeTokenCredential' in earlier PR. we can use the same class, I think.